### PR TITLE
Adds rocket.chat

### PR DESCRIPTION
**Pull request by pierreozoux**

### CODE REVIEW

**Comment by tianon** [comment]
Looks sane, but you'll want to check out the "Details" link for Travis and fix the issues listed there. :+1:

File: rocket.chat/content.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+It is a great solution for communities and companies wanting to privately host their own chat service or for developers looking forward to build and evolve their own chat platforms.
+
+%%LOGO%%
+
+# How to use this image
+
+First, start an instance of mongo:
+
+    docker run --name db -d mongo mongod --smallfiles
```
Comment (yosifkit) [inline rocket.chat/content.md]: Shouldn't need `mongod`, the mongo image just works when you pass in flags :smile:.

Comment (pierreozoux) [inline rocket.chat/content.md]: Good catch, thanks!

**Comment by pierreozoux** [comment]
Fixed!

**Comment by yosifkit** [comment]
LGTM

**Comment by tianon** [comment]
Heh, can you rename `logo.png` to `logo.svg`?  It won't render right now, but IMO we should adapt `update.sh` to handle SVGs directly instead of just rasterizing that!

**Comment by pierreozoux** [comment]
@tianon done.

**Comment by tianon** [comment]
:heart:! LGTM

**Pull request merged**